server/schedule/operator_controller.go

```diff
@@ -65,6 +65,7 @@ type OperatorController struct {
 	hbStreams       *hbstream.HeartbeatStreams
 	fastOperators   *cache.TTLUint64
 	histories       *list.List
+	records         *list.List
```

Comment:
Why not combine them by recording the status?

Reply:
`histories` only records the successful operators and use them to describe the scheduler trend on stores. 
And it records `add-peer`, `remove-peer ` and `transfer-leader` steps not operaoter dimensionality.